src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -69,6 +74,17 @@ public void setNotes(ArrayList<Note> notes) {
         this.notes = notes;
     }
 
+    public ArrayList<Note> archiveNotes() {
+
+        for (int i = 0; i < notes.size(); i++) {
+            if (notes.get(i).getIsArchived()) {
+                archivedNotes.add(notes.get(i));
+            }
+        }
+
+        return archivedNotes;
+    }
+
```

Comment:
This function shld be split into 2. 1 is getArchivedNotes() where it just return the arrayList. Another function is archiveNote(int index), overloaded with archieveNote(string title). ArchiveNoteCommand will call this function and this function will handle will removing of note from the original list and adding into the archivedNotes arrayList

Reply:
Likwise there should be an function for unarchiveNote(int index), unarchiveNote(string title)